RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h

```diff
@@ -49,7 +88,9 @@ class EcalClusterLazyToolsBase {
   // get BasicClusterSeedTime of the seed basic cluser of the supercluster
   float SuperClusterSeedTime(const reco::SuperCluster &cluster);
   // get BasicClusterTime of the seed basic cluser of the supercluster
-  float SuperClusterTime(const reco::SuperCluster &cluster, const edm::Event &ev);
+  inline float SuperClusterTime(const reco::SuperCluster &cluster, const edm::Event &ev) {
```

Comment:
this is the only change that's somehow not related to the ESGetToken migration. Did you check the impact on CPU time & code size?

Reply:
Hi, no I didn't check that. Why do you expect CPU time to increase? This is a very simple function, so I thought it might be more appropriate to define it in the header as `inline`. In theory this should actually save CPU time because there is not function call, but the main reason why I did it was to simplify the code by not separating declaration and definition of this function. And code size should also not increase I think, as the inlined function is just a function call itself.

Reply:
OK, thanks for the clarification. I don't expect any slowdown from inlining of course, I was just curious as to the motivation, which you now explained.

Reply:
(if anything, I was expecting a speedup, which I figured was the motivation for this change)